src/parser/shared/modules/spells/bfa/essences/TheWellOfExistence.js

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS/index';
+import { formatNumber, formatPercentage } from 'common/format';
+import SpellLink from 'common/SpellLink';
+
+import ItemHealingDone from 'interface/others/ItemHealingDone';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import ItemStatistic from 'interface/statistics/ItemStatistic';
+import StatisticGroup from 'interface/statistics/StatisticGroup';
+
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import Abilities from 'parser/core/modules/Abilities';
+
+const debug = false;
+
+// Can these values be queried? 
```

Comment:
Can these be queried? Especially the double healing cap, seems dumb to have these magic numbers if these values might be re-balanced by blizz one day.

Reply:
in general, no. its an unfortunate maintenance burden every time there is a hotfix/patch

Reply:
updated. now queries